src/coreclr/pal/inc/pal_mstypes.h

```diff
@@ -147,19 +125,6 @@ extern "C" {
 // Misc. type helpers
 ////////////////////////////////////////////////////////////////////////
 
-#ifdef _MSC_VER
-
-// MSVC's way of declaring large integer constants
-// If you define these in one step, without the _HELPER macros, you
-// get extra whitespace when composing these with other concatenating macros.
-#define I64_HELPER(x) x ## i64
-#define I64(x)        I64_HELPER(x)
```

Comment:
There is `INT64_C` macro according to cppreference. Not sure of it's feasibility.

Reply:
The `INT64_C` macro applies the "at least" logic and relying on the vagaries of a particular implementation with respect to "at least" is not something I think we should adopt.